### ISSUE TITLE
Wallet saving fails on Windows

### ISSUE THREAD

**Issue by zlalvani** [NebulousLabs/Sia#382]
Running on 32-bit cygwin: 
```
$ siad
Warning: config file not found. Default values will be used.
Failed to create daemon: rename sia.wallet_temp sia.wallet: Cannot create a file when that file already exists.
```
Removing the files manually just throws the same error.

**Comment by seveibar** [comment]
We had siad previously working perfectly- so we're not sure why it's broken now

**Comment by lukechampine** [comment]
This appears to be a Windows-specific problem: https://code.google.com/p/go/issues/detail?id=3366

A fix is due for Go 1.5, but in the meantime we should switching from `os.Rename` to `os.Copy`.

**Comment by DavidVorick** [comment]
Would it be enough to call a remove=>copy on the original file? It's no longer atomic, which means we'll need the primary and the backup.

Will copy do the remove automatically on Windows?

And perhaps we should have behavior that looks at the temp file if the primary file is not found.
